### PR TITLE
Persistence Component: expose input files bytes size for Snowflake BulkLoad 

### PR DESCRIPTION
**Pull request by rengam32**
What type of PR is this?
Improvement

What does this PR do / why is it needed ?
Added the support to return input files bytes metrics for Snowflake during bulk load in IngestorResult

Which issue(s) this PR fixes:
Other notes for reviewers:
Change proposed by @prasar-ashutosh

Does this PR introduce a user-facing change?
Only impacts users of legend-engine-xts-persistence-component

### CODE REVIEW

File: legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/jdbc/SnowflakeJdbcTransactionManager.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        }
+        catch (SQLException e)
+        {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public TabularData convertResultSetToTabularData(String sql, int rows)
+    {
+        try
```
Comment (prasar-ashutosh) [inline legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/jdbc/SnowflakeJdbcTransactionManager.java:63]: We do not need the outside try and let it throw the exception that JDBCHelper can catch

File: legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/jdbc/SnowflakeJdbcTransactionManager.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+public class SnowflakeJdbcTransactionManager extends JdbcTransactionManager
+{
+    public SnowflakeJdbcTransactionManager(Connection connection) throws SQLException
+    {
+        super(connection);
+    }
+
+    public TabularData convertResultSetToTabularData(String sql)
+    {
+        try
```
Comment (prasar-ashutosh) [inline legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/jdbc/SnowflakeJdbcTransactionManager.java:38]: We do not need the outside try and let it throw the exception that JDBCHelper can catch

File: legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/SnowflakeSink.java [hunk 3]
```diff
@@ -71,6 +73,11 @@
 import org.finos.legend.engine.persistence.components.relational.snowflake.sql.visitor.BatchEndTimestampVisitor;
 import org.finos.legend.engine.persistence.components.relational.snowflake.sql.visitor.CastFunctionVisitor;
 import org.finos.legend.engine.persistence.components.relational.snowflake.sql.visitor.ClusterKeyVisitor;
+import org.finos.legend.engine.persistence.components.relational.snowflake.sql.visitor.CopyVisitor;
+import org.finos.legend.engine.persistence.components.relational.snowflake.sql.visitor.DatasetAdditionalPropertiesVisitor;
+import org.finos.legend.engine.persistence.components.relational.snowflake.sql.visitor.DigestUdfVisitor;
+import org.finos.legend.engine.persistence.components.relational.snowflake.sql.visitor.FieldVisitor;
```
Comment (kumuwu) [inline legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/SnowflakeSink.java:79]: Looks redundant - should have already been imported

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**